### ISSUE TITLE
Bones clipping thru collision

### ISSUE THREAD

**Issue by Platonic-DIVA** [shteeve3d/blender-wiggle-2#33]
Bones don't collide with collision objects, but pass through them in a single frame

**Comment by Webslug** [comment]
I have the same issue.

**Comment by TheRealA77** [comment]
yeah i also noticed something similar. Collisions tend to get really janky in general, for some reason
